src/index.ts

```diff
@@ -33,8 +34,12 @@ function getServiceBaseUrl(config: Config) {
 function makeBlobServiceClient(config: Config) {
     const account = trimParam(config.account);
     const accountKey = trimParam(config.accountKey);
+    const sasToken = trimParam(config.sasToken);
     const serviceBaseURL = getServiceBaseUrl(config);
-
+    //if accountKey doesn't contain value return below line
+    if (sasToken != '') {
+        return BlobServiceClient.fromConnectionString(`${serviceBaseURL}?${sasToken}`);
```

Comment:
When adding a sasToken I get an error here

the sasToken has a format of `?sv=....` 

Are you looking to use a connection string vs a sasToken?

Reply:
Yeah